DBDebugToolkit/Classes/Location/DBLocationTableViewController.m

```diff
@@ -152,15 +154,17 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
             UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:customLocationViewController];
             [self.navigationController presentViewController:navigationController animated:YES completion:nil];
         }
-        } else if (indexPath.section == 1) {
-            self.selectedIndex = @(indexPath.row);
-            NSMutableArray *presetLocations = self.locationToolkit.gpxFilesLocations[indexPath.row]; // passed gpx files array
-            //DBPresetLocation *presetLocation;
-            self.locationToolkit.simulatedLocation = presetLocations;//[[CLLocation alloc] initWithLatitude:presetLocation.latitude
-            //longitude:presetLocation.longitude];
-            self.resetButton.enabled = YES;
-            [self.tableView reloadData];
-
+    }
+    else if (indexPath.section == 1) {
+        self.selectedIndex = @(indexPath.row);
+        NSMutableArray *presetLocations = self.locationToolkit.gpxFilesLocations[indexPath.row]; // passed gpx files array
```

Comment:
Is it ever possible that `gpxFilesLocations` could be empty?  If so there are some cases of `firstObject` that will fail.